vxquery-xtest/src/test/resources/TestSources/json/object/nested_object_array.json

```diff
@@ -0,0 +1 @@
+[{"1":{"2":[3,{"4":5}]}},1]
```

Comment:
Add a test case that has a list of objects with an internal array.

Ex: [{"1a":{"2a":[3,{"4a":5}]}},1,{"1b":{"2b":[3,{"4b":5}]}},{"1c":{"2c":[3,{"4c":5}]}}]